packages/eventive/src/eventive.ts

```diff
@@ -96,7 +106,11 @@ export function eventive<
   type Output = Eventive<DomainEvent, State>;
 
   const eventsCollection = options.db.collection<DomainEvent>(
-    options.dbCollectionName ?? "events"
+    options.dbEventsCollectionName ?? "events"
+  );
+  const snapshotsCollection = options.db.collection<BaseEntity<State>>(
+    options.dbSnapshotsCollectionName ??
+      `${snakeCase(options.entityName)}_snapshots`
```

Comment:
Does 'options.entityName' require a validation?

It would be very edge case but this situation is possible yet like below.

i.e. `user_name`, `userName` use same collection name.

Reply:
in that case usually user should define db collection name explicitly